code/api/src/config/database.json

```diff
@@ -1,14 +1,14 @@
 {
   "development": {
-    "username": "postgres",
+    "username": "michaelalexlynch",
     "password": null,
```

Comment:
do we want to keep this username as `postgres` so that it works the same for all of us?